src/client/components/Purchases/Purchases.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Purchases from './Purchases.component';
+
+export default { title: 'Components / Purchases', component: Purchases };
+
+// eslint-disable-next-line react/jsx-props-no-spreading
+const Template = (args) => <Purchases {...args} />;
+
```

Comment:
hi @Sarulathaanbu can you give more args values for the Purchases object, since you are passing only one value for purchase, its not sure that if you have more purchase orders, then how they looks and how they are aligned? Fx. you can give array of this  object {orderId: '12398098',date: '30/09/2021'} with different values.